### DISCUSSION TITLE
Neighbor

### DISCUSSION THREAD

**Discussion by svernierAI** [before-interop/anomalieAdresse#44] [General]
Bonjour Geoffrey,

Nous avons détecté une incohérence sur le rattachement de l'objet neighbor entre : 
- AnomalieAdresseCreation dans lequel il est rattaché à l'objet Building
- AnomalieAdresseNewBuilding dans lequel il est directement rattaché à l'objet Anomalie
A choisir, le rattachement à l'objet Building.

Autre, mais plus sur la forme que le fond, les valeurs par défaut de @basetype et @type pour les objets Building et Address ne sont pas correctement définis.

Merci,
Sébastien

**Comment by ggrebert** [comment]
fixed: https://github.com/before-interop/anomalieAdresse/commit/87b9c425c44122fe80103eb8409e0af957c3eb7f
